tests/mock_test.py

```diff
@@ -4,20 +4,30 @@
 
 
 @pytest.mark.parametrize(
-    's',
+    ('s', 'args'),
     (
         pytest.param(
             'import contextlib, mock, sys\n',
+            ((3,),),
             id='does not rewrite multiple imports',
         ),
         pytest.param(
             'from .mock import patch\n',
+            ((3,),),
             id='leave relative imports alone',
         ),
+        pytest.param(
+            'from mock import patch\n'
+            '\n'
+            'patch("func")',
+            ((3,), True),
+            id='does not rewrite if no_mock==True',
+        ),
+
     ),
 )
-def test_mock_noop(s):
-    assert _fix_py3_plus(s, (3,)) == s
+def test_mock_noop(s, args):
+    assert _fix_py3_plus(s, *args) == s
```

Comment:
I would make another test probably, this is trying too hard to abuse parametrize imo

Reply:
Sure. FWIW my fork of pytest doesn't consider this abuse

Reply:
your, _more serious_ fork?

Reply:
Yes I've been up all night redeyed working on it

Reply:
Based on your comments I can see you're greeneyed with envy